saritasa_invocations/_config.py

```diff
@@ -113,11 +161,118 @@ class AlembicSettings:
     )
 
 
+class PostgresSettings:
+    """Settings for postgres module."""
+
+    load_dump_command: str = (
+        "PGPASSWORD={password} "
+        "psql "
+        "{additional_params} "
+        "--dbname={dbname} "
+        "--host={host} "
+        "--port={port} "
+        "--username={username} "
+        "--file={file}.sql"
+    )
+    dump_filename: str = "local_db_dump"
+    load_additional_params: str = "--quite"
+    dump_command: str = (
+        "PGPASSWORD={password} "
+        "pg_dump "
+        "{additional_params} "
+        "--dbname={dbname} "
+        "--host={host} "
+        "--port={port} "
+        "--username={username} "
+        "--file={file}.sql"
+    )
+    dump_additional_params: str = "--no-owner"
+
+
+# This mapping should not be filled manually. You just need create an instance
+# of `K8SDataBaseSettings` and it will be auto added to this mapping
+K8S_CONFIGS: dict[str, "K8SSettings"] = {}
```

Comment:
```suggestion
_K8S_CONFIGS: dict[str, "K8SSettings"] = {}
```
As this config is not supposed to be used outside of this module